build/cross-images/x86_64-unknown-linux-gnu.dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM evanxg852000/cross-x86_64-unknown-linux-gnu:latest
+FROM quickwit/cross-base:x86_64-unknown-linux-gnu
```

Comment:
I see three repositories on [DockerHub](https://hub.docker.com/repositories): `cross`, `cross-base` and `cross-builder`. Can we consolidate everything in `cross` and remove the other two repositories. `Cross.toml` already references imagines `quickwit/cross`.

Reply:
`cross-builder` can be safely removed it's the old one, I think it's best you remove it. 
`quickwit/cross-base` exists as an intermediate repository where we push slightly modified base images of `rust-embemded/cross` and others. I thought it's best to have them in a separate repository since we have cloned the git repos.